modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/notification-sidebar/index.scss

```diff
@@ -0,0 +1,65 @@
+.notification-container {
+	.dotted-line:after {
+		background-image: linear-gradient(
+			90deg,
+			#a3a3a3,
+			#a3a3a3 30%,
+			transparent 30%,
+			transparent 100%
+		);
+		background-size: 8px 1px;
+		content: '';
+		height: 1px;
+		position: absolute;
+		right: 15px;
+		width: 91%;
+	}
+
+	.post-container {
+		padding-left: 20px;
+
+		a {
+			cursor: pointer;
+			padding-left: 20px;
+		}
+
+		a:hover {
+			text-decoration: none;
+		}
+
+		a p {
+			color: rgba(39, 40, 51, 0.7);
+			display: flex;
+			font-size: 0.75rem;
+			font-weight: normal;
+			margin-bottom: 7px;
+			margin-top: 0;
```

Comment:
```suggestion
		
```
I believe you're already doing this with bootstrap class mt-0

Reply:
Done!